### PR TITLE
Add remaining samples P1 bindings/triggers

### PR DESCRIPTION
**Pull request by JunyiYi**
**Remarks**
The verified bindings and triggers are added in #38 . In this PR, we will focus on the following content.

**Not tested**
- EventGridTrigger: Requires deployment to Azure, not testable locally.

**Failed**
- CosmosDBTrigger, CosmosDBInput, CosmosDBOutput: Error message is `"Unable to load DLL 'Microsoft.Azure.Documents.ServiceInterop.dll' or one of its dependencies"`.

**Related issues**
#34 , #36 .

### CODE REVIEW

**Comment by rloutlaw** [comment]
I just looked at the Cosmos DB Output example again. Can you test this function to make sure it matches the description in the JavaDocs? They code I'm reading and the text in the description seem to do two different things, if this isn't the case is there a more clear/concise way to construct the sample so that the intent of the code is clear?

**Comment by JunyiYi** [comment]
Sure @rloutlaw . But the `CosmosDB` related bindings are failing in the production environment. And we have no ETA for that.

**Comment by JunyiYi** [comment]
Hi @rloutlaw , I have updated the code according to your suggestion, please take a look. Thanks.

**Comment by pragnagopa** [comment]
@JunyiYi - Can you please verify with the latest bits? I will merge this in one you confirm.

File: src/main/java/com/microsoft/azure/functions/annotation/CosmosDBTrigger.java [hunk 1]
```diff
@@ -21,18 +21,20 @@
...
- *                      leaseCollectionName = "",
- *                      connectionStringSetting = "myconnvarname") MyDataItem[] items,
+ * <pre>{@literal @}FunctionName("cosmosDBMonitor")
+ * public void cosmosDbLog(
+ *    {@literal @}CosmosDBTrigger(name = "database",
+ *                      databaseName = "ToDoList",
+ *                      collectionName = "Items",
+ *                      leaseCollectionName = "leases",
+ *                      createLeaseCollectionIfNotExists = true,
+ *                      connectionStringSetting = "AzureCosmosDBConnection") String[] items,
```
Comment (pragnagopa) [inline src/main/java/com/microsoft/azure/functions/annotation/CosmosDBTrigger.java]: Input as String[]  does this work? What is the sample data you use to trigger this function?

Comment (JunyiYi) [reply src/main/java/com/microsoft/azure/functions/annotation/CosmosDBTrigger.java]: I just guessed the type. That's why we need to confirm with the latest bits.

**Pull request merged**